app/src/main/java/dev/arkbuilders/arkmemo/data/ResourceMeta.kt

```diff
@@ -1,12 +1,18 @@
 package dev.arkbuilders.arkmemo.data
 
+import android.os.Parcelable
 import dev.arkbuilders.arklib.ResourceId
+import kotlinx.parcelize.IgnoredOnParcel
+import kotlinx.parcelize.Parcelize
 import java.nio.file.attribute.FileTime
+import java.time.Instant
 
+@Parcelize
 data class ResourceMeta(
     val id: ResourceId,
     val name: String,
     val extension: String,
-    val modified: FileTime,
+    @IgnoredOnParcel
+    val modified: FileTime = FileTime.from(Instant.now()),
     val size: Long
```

Comment:
This data structure doesn't make sense to me...
* `name` and `extensions` are only needed to map `ResourceId` and `File` used for storage, this job is carried by index, so we should just use it
* do we really need `modified`? if we need we should not take it from the filesystem, we should take it from the index
* `size` is a part of `id`, so no need to duplicate this field

Reply:
I think it's time we replace this with meta from arklib